numpy/lib/function_base.py

```diff
@@ -1974,7 +1974,7 @@ class vectorize:
     References
     ----------
     .. [1] NumPy Reference, section `Generalized Universal Function API
-           <https://docs.scipy.org/doc/numpy/reference/c-api.generalized-ufuncs.html>`_.
+           <https://numpy.org/doc/stable/reference/c-api/generalized-ufuncs.html>`_.
```

Comment:
This feels like we should be using an internal link.

Reply:
I'm surprised not to see this case covered in the [docstring guide](https://numpy.org/devdocs/docs/howto_document.html#numpydoc-docstring-guide).

My thought was that content of the .py files should be self-contained.  

If we link internally I'll need a markup suggestion.